ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChart.ts

```diff
@@ -14,13 +14,15 @@
 import { PanelPlugin } from '@perses-dev/plugin-system';
 import { createInitialTimeSeriesChartOptions, TimeSeriesChartOptions } from './time-series-chart-model';
 import { TimeSeriesChartOptionsEditor } from './TimeSeriesChartOptionsEditor';
+import { TimeSeriesChartOptionsEditorSettings } from './TimeSeriesChartOptionsEditorSettings';
 import { TimeSeriesChartPanel } from './TimeSeriesChartPanel';
 
 /**
  * The core TimeSeriesChart panel plugin for Perses.
  */
 export const TimeSeriesChart: PanelPlugin<TimeSeriesChartOptions> = {
   PanelComponent: TimeSeriesChartPanel,
-  OptionsEditorComponent: TimeSeriesChartOptionsEditor,
+  PanelQueryEditorComponent: TimeSeriesChartOptionsEditor,
+  panelOptionsEditorComponents: [{ id: 'settings', label: 'Settings', content: TimeSeriesChartOptionsEditorSettings }],
```

Comment:
Should panelOptionsEditorComponents be capitalized like the others and if it is an array, what is an example of when you’d need to add more than one component inside the array?

Reply:
> Should panelOptionsEditorComponents be capitalized like the others

I didn't capitalize it because panelOptionsEditorComponents is an array whereas the others indicate that it should be React.ComponentType.

> what is an example of when you’d need to add more than one component inside the array?

I don't have a specific example but this gives the flexibility to plugin authors if they ever want to have multiple tabs

Reply:
We haven't built it yet, but I remember there being a hypothetical example for a table plugin that might split general settings and column settings into two tabs in a case like this.